docs/connecting-to-a-database.md

```diff
@@ -0,0 +1,97 @@
+# Connecting to a database
+
+> ⚠️ **Warning**: Particularly if you're connecting to a production database,
+> proceed with caution and look to pair with someone else.
```

Comment:
Love this!

Reply:
Stolen idea from a doc Chris made somewhere for the frontend. 😜